regen-network/regen-web
# feat: credit class issuance and info table

## Description

Closes: regen-network/regen-registry#1723



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

1. See https://deploy-preview-1985--regen-registry.netlify.app/credit-classes/C02 and scroll down to the table.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
